eland/etl.py

```diff
@@ -479,8 +477,7 @@ def csv_to_eland(  # type: ignore
         "float_precision": float_precision,
         "na_filter": na_filter,
         "delim_whitespace": delim_whitespace,
-        "warn_bad_lines": warn_bad_lines,
-        "error_bad_lines": error_bad_lines,
+        "on_bad_lines": on_bad_lines,
```

Comment:
If we detect pandas <1.3.0 let's transform `on_bad_lines="warn"` -> `kwargs["warn_bad_lines"]=True` and `on_bad_lines="error"` -> `kwargs["error_bad_lines"] = True`